tests/xml_file_loader_tests.cc

```diff
@@ -31,18 +31,23 @@ void XMLFileLoaderTests::SetUp() {
   solver_control_file = "solvercontrol.xml";
   CreateTestInputFile(solver_control_file, ControlSequenceWithSolver());
 
+  epsFile = "eps.xml";
```

Comment:
Variable names should not be camel cased: https://google.github.io/styleguide/cppguide.html#Variable_Names